README.md

```diff
@@ -5,11 +5,7 @@
 Most of the code here has been extracted from `Snuba` so that it can be reused in `Sentry` and other services.
 
 Arroyo provides:
+
 * Consumer and producer interfaces. The primary use case is for working with Apache Kafka streams, however it also supports custom backends and includes local (memory or file based) consumer and producer implementations
 * Consumer strategy interface that helps build the processing strategy for how raw messages that are consumed should be filtered, transformed, batched and flushed as required
 * A synchronized consumer implementation that supports following another consumer by only consuming messages once already consumed and committed by another consumer group
-
-
-# License
-
-Licensed under the Apache-2.0 license, see [`LICENSE`](https://github.com/getsentry/arroyo/blob/main/LICENSE)
```

Comment:
We don't need this. It also creates a potential for divergence in case of a license change in the future.